### ISSUE TITLE
Cluttered root-directory

### ISSUE THREAD

**Issue by hoytnix** [hoytnix/hoyt.io#3]
As of 2a2e963 there are 28-files in the root-directory:
- 6 of which should be required in the application (app-files / version-control).
- 12 of which belong to various continuous-integration services.
- 4 of which belong to Docker
- 3 of which belong to Firebase
- 3 of which belong to the asset pipeline (Grunt / Node)

```
# Application
.gitignore  Integrated Jacebrowning's CI; removed pytest for right now  2 hours ago
LICENSE     first commit    18 days ago
README.md   Fixed link to CircleCI  2 hours ago
alembic.ini     first commit    18 days ago
requirements.txt    Pytest works now    33 minutes ago
setup.py    Fixed checker errors    a minute ago

# Continuous-Integration
.coveragerc     Fixed checker errors    a minute ago
.pep257     Integrated Jacebrowning's CI; removed pytest for right now  2 hours ago
.pylintrc   Fixed Pylint errors     3 hours ago
.pyup.yml   Integrated Jacebrowning's CI; removed pytest for right now  2 hours ago
.style.yapf     yapf'd everything   19 hours ago
.travis.yml     Fixed CI-service settings   2 hours ago
circle.yml  Fixed CI-service settings   2 hours ago
Makefile    Fixed checker errors    a minute ago
pytest.ini  Pytest works now    33 minutes ago
scent.py    Integrated Jacebrowning's CI; removed pytest for right now  2 hours ago
requirements-ci.txt     Removed unused requirements     2 hours ago
requirements-dev.txt    Pytest works now    33 minutes ago

# Docker
.dockerignore   first commit    18 days ago
.env    first commit    18 days ago
Dockerfile  first commit    18 days ago
docker-compose.yml  first commit    18 days ago

# Firebase
.firebaserc     first commit    18 days ago
database.rules.json     first commit    18 days ago
firebase.json   first commit    18 days ago

# Asset Pipeline
Gruntfile.coffee    Added is_published flag to Posts    19 hours ago
fontello.json   Reusable elements for blog  12 days ago
package.json    Templates restructure, blog features, grunt-fontello    16 days ago
```

**Comment by hoytnix** [comment]
The following can be bundled in a single `setup.cfg`:
- `.coveragerc`
- `.pep257`
- `.style.yapf`
- `pytest.ini`

This would reduce the file-count from 28 to 25.

**Comment by hoytnix** [comment]
The purpose of the `Dockerfile` has been deprecated by the _watch_-function of the Makefile, so both files `Dockerfile` and ``.dockerignore` may be removed. (-2)

The `.env`-file may be merged with `docker-compose.yml` because the contents do not need to be secret. (-1)

That would bring the count down from 25 to 22.

**Comment by hoytnix** [comment]
These two changes can be seen in d57e8dcbe9ad644fc7d80a87ce2b859a32c68201. 

For the next change I would like to look into moving everything related to Firebase and the asset-pipeline into a single-folder, potentially named `static`.

**Comment by hoytnix** [comment]
In bcf10c5 all of the Docker-files were moved to the `database`-folder.

The root-directory currently has 21 files and 7 directories.

**Comment by hoytnix** [comment]
In e3a6e6c9ff22a0163580c10378a55b9f98bf8408 I moved everything related to Firebase and the asset-pipeline to the directory `static-server` as I suggested before.

The root-directory currently has 15 files and 7 directories.

**Comment by hoytnix** [comment]
I believe this has been solved in edc5a62. The three main components: the database, application, and static server; have been decoupled into their own directories.

The root-directory currently has 8 files (-20), and 3 directories (-3).
